src/main/java/soconnect/model/ModelManager.java

```diff
@@ -26,6 +27,8 @@ public class ModelManager implements Model {
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
     private final FilteredList<Todo> filteredTodos;
+    private final ObservableList<Tag> tags;
```

Comment:
Are you using this? I'm pretty sure @ChongCheeKaiClarence already removed it

Reply:
Oh I don't need it, I think I didn't remove this properly when resolving the merge conflict. Thank!